index.html

```diff
@@ -2375,16 +2375,33 @@ <h2>
         |params|.
         </li>
       </ol>
+    </section>
+    <section>
+      <h2>
+        Launch queue and launch params
```

Comment:
The 
```
      <p>
        A {{Window/window}} has an associated {{LaunchConsumer}} <dfn>assigned
        launch consumer</dfn>.
      </p>
      <p>
        A {{Window/window}} has an associated {{LaunchParams}} <dfn>unconsumed
        launch params</dfn>.
      </p>
```
definitions should probably be moved out of the file handling section and into this one.